sound/soc/sof/intel/hda.c

```diff
@@ -26,7 +26,6 @@
 #include <sound/intel-nhlt.h>
 #include <sound/sof.h>
 #include <sound/sof/xtensa.h>
-#include <trace/events/sof_intel.h>
```

Comment:
this needs to go with the previous patch that added this twice

Reply:
good catch, just fixed with latest push.